doc/setting_up_vcpkg.md

```diff
@@ -21,17 +21,6 @@ The `use-prov-client` feature uses the syntax: `vcpkg install azure-iot-sdk-c[us
     -Duse_prov_client=ON
```

Comment:
should `use-prov-client` be removed from the docs as well?

Reply:
Yes, good catch, thank you!